pyaerocom/io/__init__.py

```diff
@@ -93,6 +93,7 @@ def coda_available():
 from .read_ghost import ReadGhost
 
 from .readgridded import ReadGridded, ReadGriddedMulti
+from .read_mscw_ctm import ReadEMEP, ReadMscwCtm
```

Comment:
```suggestion
from .read_mscw_ctm import ReadMscwCtm
```

would need to make sure ReadEMEP is not imported across the lib modules...